src/components/atoms/OcDrop/OcDrop.vue

```diff
@@ -206,7 +205,8 @@ export default {
   line-height: inherit;
 
   .tippy-content {
-    padding: var(--oc-space-xsmall);
+    // note: needed so that the box shadow from `oc-box-shadow-medium` doesn't get suppressed
+    padding: 8px;
```

Comment:
Please not that this will break acceptance tests in `web` because they close the OcDrop by clicking outside by a hardcoded pixel value 😉

Reply:
You mean as an additional comment in the code? or just for me when updating in web? ;-)

Reply:
Just keep it in mind for the web update ;) acceptance test fix is in this commit https://github.com/owncloud/web/commit/c959579d8c2533aaa28842cc357d646b6dae5a06